django/contrib/admin/views/main.py

```diff
@@ -155,6 +158,7 @@ def get_filters(self, request):
                 # remove duplicate results.
                 if lookup_params_count > len(lookup_params):
                     use_distinct = use_distinct or lookup_needs_distinct(self.lookup_opts, field_path)
+                    has_active_filters = True
```

Comment:
Not sure if my thinking if correct here, but it seems like we need to handle the `if callable(list_filter):` branch as well. I'm thinking of setting `has_active_filters = True` after the `else` and... and... blah blah!

<img width="762" alt="django-contrib___Volumes_Projects_contrib_django-contrib__-__Volumes_Projects_contrib_django_django_contrib_admin_views_main_py__django_" src="https://user-images.githubusercontent.com/1173748/82323860-f2941c80-99d8-11ea-8fbe-5303a28bcde1.png">

Reply:
Good catch :+1: I fixed this and added extra tests.